crates/re_log_types/src/time.rs

```diff
@@ -85,7 +89,17 @@ impl Time {
             } else {
                 time::format_description::parse(&date_format).unwrap()
             };
-            datetime.format(&parsed_format).unwrap()
+
+            // Return in the local timezone.
+            if will_show_timestamps_in_local_timezone {
+                let local_datetime = datetime.to_offset(local_offset.unwrap());
+                return local_datetime.format(&parsed_format).unwrap();
+            }
+
+            // Return in UTC.
+            let mut formatted_datetime = datetime.format(&parsed_format).unwrap();
+            formatted_datetime.push('Z');
+            formatted_datetime
```

Comment:
```suggestion
            if let Some(local_offset) = local_offset {
                // Return in the local timezone.
                let local_datetime = datetime.to_offset(local_offset);
                local_datetime.format(&parsed_format).unwrap()
            } else {
                // Return in UTC.
                format!("{}Z", datetime.format(&parsed_format).unwrap())
            }
```

Reply:
same in the other formatting place

Reply:
Thanks! I think we also need an `Ok(` in there, i.e. `if let Some(Ok(local_offset)) = local_offset {`, correct? Because `Some(` will just unwrap the `Option` but not check the `Result`?

<img width="976" alt="Screenshot 2023-09-29 at 2 20 08 PM" src="https://github.com/rerun-io/rerun/assets/1396242/ec234feb-9fa8-46fa-a102-56f96aeca1af">